internal/engine/local_container_build_and_deployer.go

```diff
@@ -25,7 +26,7 @@ type LocalContainerBuildAndDeployer struct {
 	k8sClient k8s.Client
 	analytics analytics.Analytics
 
-	deployInfo   map[model.ManifestName]k8s.ContainerID
+	deployInfo   map[reference.NamedTagged]k8s.ContainerID
```

Comment:
i don't think we want to use a NamedTagged as a map key. 

there are multiple internal representations that satisfy this interface. Go will use the internal representation as the key, not the NamedTagged part of it.

we probably want to use a custom struct with a Name and a Tag field

Reply:
good call, will do